packages/apps/helpdesk/src/screens/TicketFormScreen.js

```diff
@@ -16,267 +16,100 @@
+import {useSelector, useDispatch, FormView} from '@axelor/aos-mobile-core';
 import {createTicket, updateTicket} from '../features/ticketSlice';
-import {
-  ContactPartnerSearchBar,
-  CustomerSearchBar,
-  DurationFormInput,
-  ProgressFormInput,
-  ProjectSearchBar,
-  TicketTypeSearchBar,
-  UserSearchBar,
-} from '../components';
-import {Ticket} from '../types/';
-
-const PROJECT_KEY = 'project';
-const CUSTOMER_KEY = 'customerPartner';
-const CONTACT_KEY = 'contactPartner';
-
-const DEFAULT_TICKET_VALUE = {duration: 0};
-
-const isObjectMissingRequiredField = object => checkNullString(object?.subject);
+import {updateCustomer} from '../features/customerSlice';
 
 const TicketFormScreen = ({navigation, route}) => {
   const idTicket = route?.params?.idTicket;
-  const dispatch = useDispatch();
-  const I18n = useTranslator();
-  const Colors = useThemeColor();
+  const _dispatch = useDispatch();
 
   const {ticket} = useSelector(state => state.ticket);
 
-  const [_ticket, setTicket] = useState(
-    idTicket != null ? ticket : DEFAULT_TICKET_VALUE,
-  );
-  const [disabledButton, setDisabledButton] = useState(
-    isObjectMissingRequiredField(_ticket),
-  );
-
-  const handleTicketFieldChange = useCallback((newValue, fieldName) => {
-    setTicket(current => {
-      if (current?.[fieldName] === newValue) {
-        return current;
-      }
-
-      const updatedObject = current == null ? {} : {...current};
-
-      updatedObject[fieldName] = newValue;
-
-      if (fieldName === PROJECT_KEY) {
-        updatedObject[CUSTOMER_KEY] = newValue?.clientPartner;
-        updatedObject[CONTACT_KEY] = newValue?.contactPartner;
-      }
-
-      setDisabledButton(isObjectMissingRequiredField(updatedObject));
-
-      return updatedObject;
-    });
-  }, []);
-
-  const updateTicketAPI = useCallback(() => {
-    dispatch(
-      updateTicket({
-        ticket: _ticket,
-      }),
-    );
-
-    navigation.navigate('TicketDetailsScreen', {
-      idTicket: _ticket.id,
-    });
-  }, [dispatch, _ticket, navigation]);
-
-  const createTicketAPI = useCallback(() => {
+  const defaultValue = useMemo(() => {
+    const _default = {
+      duration: 0,
+    };
+    if (idTicket != null) {
+      return {
+        ..._default,
+        id: ticket?.id,
+        version: ticket?.version,
+        subject: ticket?.subject,
+        progressSelect: ticket?.progressSelect,
+        project: ticket.project,
+        customerPartner: ticket?.customerPartner,
+        contactPartner: ticket?.contactPartner,
+        ticketType: ticket?.ticketType,
+        prioritySelect: ticket?.prioritySelect,
+        startDateT: ticket?.startDateT,
+        endDateT: ticket?.endDateT,
+        deadlineDateT: ticket?.deadlineDateT,
+        duration: ticket?.dureation,
+        assignedToUser: ticket?.assignedToUser,
+        responsibleUser: ticket?.responsibleUser,
+        description: ticket?.description,
+      };
```

Comment:
you can just do a copy of ticket and reset tricky values like duration using defaultValue : 

```
      {
        ...ticket,
        duration: ticket.duration || _default.duration,
      };
```

Also caution when using field name, "dureation" does not exist